rmf_fleet_adapter_python/README.md

```diff
@@ -45,7 +45,18 @@ You may then probe the effects on the ROS2 graph by subscribing to the following
 - `/dispenser_states`
 - `/task_summaries`
 
+### Running Traffic Light Example
+This will showcase an example of having 2 "traffic light" robots using RMF.
 
+```bash
+# First Terminal
+ros2 run rmf_fleet_adapter_python schedule_blockage_nodes
```

Comment:
I'm not sure if it's a typo, but every use of `blockage` should really be `blockade`.

`blockage` implies that some kind of unintentional clog exists. `blockade` implies that a zone is intentionally blocked off in order to guarantee passage for a specific party.

Reply:
Done!